### ISSUE TITLE
Barcode scanner/Bookbag not spawning

### ISSUE THREAD

**Issue by DeteledSawI** [Aurorastation/Aurora.3#11681]
So I like playing as tech support (librarian), and a few days ago I decided to reread the librarian guide to see if I could find something about the arcaneum table and realized that you were supposed to scan books to add them to the inventory. After a good search of the library, I realized that you neither spawn with a bookbag or a scanner to use.
After getting a scanner spawned for me, I linked it to the Library Desk computer and began to scan books, only to find that it wasn't sending the books to the computer inventory.

In conclusion, scanner broke, plz fix

**Comment by Chen-Yakumo** [comment]
Like Drill Technicians and Shaft Miners being alternate titles, Tech Support does not spawn with the same equipment that Librarians spawn with. Librarians spawn with a barcode scanner and bookbag, tech support spawns with a circuit kit. This is intentional.
